src/commands/latency-histogram.json

```diff
@@ -18,6 +18,30 @@
             "REQUEST_POLICY:ALL_NODES",
             "RESPONSE_POLICY:SPECIAL"
         ],
+        "reply_schema": {
+            "type": "object",
+            "description": "A map where each key is a command name, and each value is a map with the total calls, and an inner map of the histogram time buckets.",
+            "patternProperties": {
+                "^.*$": {
```

Comment:
This pattern accepts any string. It's for the command name like get,set,lpush ..

See related the change in `generate-command-code.py` to make it work while generating C files. 
Normally, we use these strings as the struct name / variable name in the generated C code. We have to replace `^ .$` as these are not valid naming characters in C.